module1.js

```diff
@@ -28,7 +32,11 @@ function question1(name1) {
 */
 
 function question2(number1, number2) {
-  // TODO:
+  if (number1 == number2) {
+    return true;
```

Comment:
Quite a few other students have also written code like:
```
if (someCondition) {
  return true;
} else {
  return false;
}
```
In these cases note that you can often just `return someCondition;`, or `return !!someCondition;` if it's not already evaluating to `true` and `false`.